reactor-core/src/main/java/reactor/core/publisher/Operators.java

```diff
@@ -1413,7 +1413,7 @@ public void subscribe(Subscriber<? super T> s) {
 		}
 	}
 
-	static final CoreSubscriber<?> EMPTY_SUBSCRIBER = new CoreSubscriber<Object>() {
+	static final Fuseable.ConditionalSubscriber<?> EMPTY_SUBSCRIBER = new Fuseable.ConditionalSubscriber<Object>() {
```

Comment:
Should not be a big deal to support ConditionalSubscriber interface which includes CoreSubscriber interface as well

Reply:
looks good to me 👍 
conceptually, `instanceof ConditionalSubscriber` should be done early on, during `subscribe()`, at which point the passed `Subscriber` is not supposed to be an EMPTY_SUBSCRIBER anyway so all good here ✅